src/pages/home/report/ReportActionsView.js

```diff
@@ -470,8 +474,9 @@ class ReportActionsView extends React.Component {
      * Hide the new MarkerBadge
      */
     hideMarker() {
-        this.setState({isMarkerActive: false}, () => {
-            this.setState({localUnreadActionCount: 0});
+        this.setState({
+            isMarkerActive: false,
+            localUnreadActionCount: 0,
```

Comment:
@parasharrajat @roryabraham do either of you know whether this callback is doing anything?

**introduced here:** https://github.com/Expensify/App/pull/4773

Reply:
I think I moved the count reset to the callback so that the user does not see an immediate reset to 0 while the MarkerBadge is hiding. Not the best way to do this as animation has a timeout but this callback will fire early.

Reply:
>so that the user does not see an immediate reset to 0 while the MarkerBadge is hiding. Not the best way to do this as animation has a timeout but this callback will fire early.

**thought:** Perhaps a better way to do that is to wait until the animation is over to set the unread action count. 

@tgolen not sure if we need to do anything here though as unclear if the original concern is something we should solve for i.e.

>so that the user does not see an immediate reset to 0 while the MarkerBadge is hiding

Reply:
IMO since the animation is short, it doesn't really matter what the count is. I agree that if it does matter, listening to the end of the animation event sounds like a better solution.